x/cronos/keeper/grpc_query.go

```diff
@@ -5,12 +5,14 @@ import (
 	"fmt"
 	"math/big"
 
+	errorsmod "cosmossdk.io/errors"
```

Comment:
maybe uses `sdkerr` ?

Reply:
we might gradually move to errorsmod instead of sdkerrors